www/templates/control/main-control.html

```diff
@@ -93,13 +93,13 @@
     </div-->
 
 
-    <!--div class="control-list-item">
```

Comment:
this is not correct - you checked in the changes to re-enable the developer zone (presumably for testing)
you need to revert them before I can push.